.expeditor/verify_private.pipeline.yml

```diff
@@ -285,7 +285,9 @@ steps:
   - label: "[integration] event-feed-service"
     command:
       - . scripts/verify_setup.sh
-      - hab studio run "source scripts/verify_studio_init.sh && start_deployment_service && chef-automate dev deploy-some chef/event-feed-service --with-deps && event_feed_integration"
+      - hab studio run "source scripts/verify_studio_init.sh &&
+          start_deployment_service &&
+          chef-automate dev deploy-some chef/data-lifecycle-service --with-deps && event_feed_integration"
```

Comment:
Super nitpicky but it seems weird to have only the last line have two commands joined with `&&`.

Reply:
I did not notice that.